news/views.py

```diff
@@ -189,12 +203,13 @@ class CreateEventView(PermissionRequiredMixin, CreateView):
         'private',
         'featured',
         'multiday',
-        'hoopla',
```

Comment:
When creating an event the `number_of_tickets` field is missing, but the information box is there. I assume the field should be there?

![image](https://user-images.githubusercontent.com/32880010/52205426-51329700-2877-11e9-98e9-1bf6b965675c.png)